sopel/plugin.py

```diff
@@ -12,6 +12,10 @@
 import functools
 import re
 
+# import and expose privileges as shortcut
```

Comment:
I really like them being shortcut, so plugin author can keep using `from sopel import plugin` to use them, while Sopel's own source code can use `sopel.privileges` instead to prevent cyclic imports.